Makefile

```diff
@@ -119,31 +122,8 @@ update_pinpoint_supported_countries:
 		> config/country_dialing_codes.yml
 	yarn normalize-yaml config/country_dialing_codes.yml config/pinpoint_supported_countries.yml
 
-lint_country_dialing_codes: update_pinpoint_supported_countries
+lint_country_dialing_codes: update_pinpoint_supported_countries ## Checks that countries supported by Pinpoint for voice and SMS are up to date
 	(! git diff --name-only | grep config/country_dialing_codes.yml) || (echo "Error: Run 'make update_pinpoint_supported_countries' to update country codes"; exit 1)
 
-check_asset_strings:
+check_asset_strings: ## Checks for strings
 	find ./app/javascript -name "*.js*" | xargs ./scripts/check-assets
-
-# ARTIFACT_DIRECTORY is the source directory containing the IDP repository that is being
-# turned into a file artifact
-build_artifact:
-	bundle config set --local cache_all true
-	bundle package
-	tar --exclude './config/agencies.yml' \
-		--exclude './config/iaa_gtcs.yml' \
-		--exclude './config/iaa_orders.yml' \
-		--exclude './config/iaa_statuses.yml' \
-		--exclude './config/integration_statuses.yml' \
-		--exclude './config/integrations.yml' \
-		--exclude './config/partner_account_statuses.yml' \
-		--exclude './config/partner_accounts.yml' \
-		--exclude './config/service_providers.yml' \
-		--exclude='./certs/sp' \
-		--exclude='./identity-idp-config' \
-		--exclude='./tmp' \
-		--exclude='./node_modules' \
-		--exclude='./geo_data/GeoLite2-City.mmdb' \
-		--exclude='./pwned_passwords/pwned_passwords.txt' \
-		--exclude='./vendor/ruby' \
-		--exclude='./config/application.yml' -cf - $(ARTIFACT_DIRECTORY) | $(GZIP_COMMAND) > $(ARTIFACT_DESTINATION_FILE)
```

Comment:
I thought about this one a bit and I think because it requires 2 env vars, it fits better as a script than as a Makefile step. In my mind, Makefile steps don't need any arguments/params/etc they just run, and this wasn't quite that

Reply:
hm, I'm not sure, I think it's ok as a Makefile task?

Reply:
although it should probably be better about requiring the variables be set

Reply:
https://github.com/18F/identity-idp/pull/5791#discussion_r779723078 <<< sorry I put this comment in another place but this was part of my thinking

Reply:
No strong preference for me one way or the other, though I tend to agree it'd be nice to set an expectation that any Makefile target could be run without additional required parameters. Would it be reasonable to at least provide defaults for them?

I sort of also agree with https://github.com/18F/identity-idp/pull/5791/files#r779732894 in that we don't really have a consistent approach for these kinds of scripts. We already have several in `bin`, `deploy`, and `scripts`. In fact, it feels like it might fit well with our other `deploy` scripts? Then again, I might not fully understand the purpose of the script in the first place. 🤷